firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/AppDistributionRelease.java

```diff
@@ -17,23 +17,44 @@
 import androidx.annotation.NonNull;
 
 /**
- * Interface for AppDistributonRelease object returned by checkForUpdate() and
+ * Data class for AppDistributionRelease object returned by checkForUpdate() and
  * updateToLatestRelease()
  */
-public interface AppDistributionRelease {
+public final class AppDistributionRelease {
```

Comment:
non-blocking nit: you may want to add `equals()` and `hashCode()`, or use an `@AutoValue` altogether.

Reply:
Good call. We will add these in a follow-up PR